boards/Makefile.include.cortexm_common

```diff
@@ -1,18 +1,34 @@
 # Target triple for the build. Use arm-none-eabi if you are unsure.
 export TARGET_TRIPLE ?= arm-none-eabi
 
-# Toolchain prefix, defaults to target triple followed by a dash, you will most likely not need to touch this.
+# Use TOOLCHAIN environment variable to select the toolchain to use.
+ifeq (,$(TOOLCHAIN))
```

Comment:
Just a thought: many build systems use TOOLCHAIN for the actual triple prefix (e.g., arm-none-eabi-). But I can't think of a better name. Opinions?